docs/generating-code-from-schema.md

```diff
@@ -3,13 +3,12 @@ The plugin generates the following:
 
 * Data types for types, input types, enums and interfaces.
 * A `DgsConstants` class containing the names of types and fields
-* Example data fetchers
```

Comment:
have anyone already removed data fetcher examples generation or is it still TODO?

Reply:
Todo, but wanted to get rid of it in the docs already